client/errors_test.go

```diff
@@ -51,6 +51,10 @@ func TestRemovePII(t *testing.T) {
 			"operation error EC2: DescribeSnapshotAttribute, https response error StatusCode: 400, RequestID: xxxx, api error InvalidSnapshot.NotFound: The snapshot 'snap-11111111111111111' does not exist.",
 			"operation error EC2: DescribeSnapshotAttribute, https response error StatusCode: 400, RequestID: xxxx, api error InvalidSnapshot.NotFound: The snapshot 'xxxx' does not exist.",
 		},
+		{
+			"AutoScalingGroup name not found - Could not find Auto Scaling group named 'hub-dev-1111'",
```

Comment:
Name leak - shouldn't include real-life names in tests

Reply:
Fixed! thanks :)